Expensify/App
# Update Add Debit Card form to match VBA styles

@luacmartins @marcaaron will you please review this?

### Details
This PR updates the add debit card form to match the form styles when adding a bank account.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5644

### Tests/QA
1. Sign into an account and go to Settings->Payments->Add Debit Card, confirm that the form fields have the same height
3. Confirm you can navigate back to the previous page
4. Submit the empty form, confirm that the fields are outlined in red (except for state field) and an error message displays below each field
5. Confirm that an alert shows at the bottom of the form, above the submit button, with a link to `fix the errors`
6. Confirm properly filling out fields in the form removes the error messages
7. Confirm you can add different values for the expiration date including MM/YYYY, MMYYYY, MM-YYYY, MM/YY, MMYY, MM-YY

### Local Testing (no QA)
7. You can use 4242424242424242 as a test debit card number to confirm the form submits properly

### Tested On


### Screenshots


| Web | Mobile |
|---|---|
| ![image](https://user-images.githubusercontent.com/3981102/138976209-a0c142c4-7b1c-4a0b-b897-d744bf9bf277.png) | ![image](https://user-images.githubusercontent.com/3981102/138976218-f0b76e72-3464-476b-8875-cadb3d3b599e.png) |
